integration-test/src/java/com/twitter/heron/integration_test/topology/serialization/resources/SerializationTopology.json

```diff
@@ -0,0 +1 @@
+[10]
```

Comment:
This is because spouts used for integration tests (com.twitter.heron.integration_test.core.IntegrationTestSpout) executes 10 times as default.